test/jdk/sun/security/provider/certpath/OCSP/OCSPNoContentLength.java

```diff
@@ -56,7 +56,7 @@ public class OCSPNoContentLength {
     static String EE_ALIAS = "endentity";
 
     // Enable debugging for additional output
-    static final boolean debug = false;
+    static final boolean debug = true;
```

Comment:
Do you intend to leave this `true`?

Reply:
Yes, I would like to leave this in the off chance this does rear its ugly head again.  I've done hundreds upon hundreds of iterations of this test both by itself and as part of tier2 runs and no failures occur.  If it did fail though, I'd like to have the extra logging.  It doesn't add very much to the output, but the info is useful.